lenses/tests/test_nslcd.aug

```diff
@@ -0,0 +1,408 @@
+  { "#comment" = "for details." }
+  {  }
+  { "#comment" = "Specifies the number of threads to start that can handle requests and perform LDAP queries." }
+  { "threads" = "5" }
+  {  }
+  { "#comment" = "The user and group nslcd should run as." }
+  { "uid" = "nslcd" }
+  { "gid" = "nslcd" }
+  {  }
+  { "#comment" = "This option controls the way logging is done." }
+  { "log" = "syslog info" }
+  {  }
+  { "#comment" = "The location at which the LDAP server(s) should be reachable." }
+  { "uri" = "ldaps://XXX.XXX.XXX" }
+  {  }
+  { "#comment" = "The search base that will be used for all queries." }
+  { "base" = "dc=XXX,dc=XXX" }
+  {  }
+  { "#comment" = "The LDAP protocol version to use." }
+  { "ldap_version" = "3" }
+  {  }
+  { "#comment" = "The DN to bind with for normal lookups." }
+  { "binddn" = "cn=annonymous,dc=example,dc=net" }
+  { "bindpw" = "secret" }
+  {  }
+  {  }
+  { "#comment" = "The DN used for password modifications by root." }
+  { "rootpwmoddn" = "cn=admin,dc=example,dc=com" }
+  {  }
+  { "#comment" = "The password used for password modifications by root." }
+  { "rootpwmodpw" = "XXXXXX" }
+  {  }
+  {  }
+  { "#comment" = "SASL authentication options" }
+  { "sasl_mech" = "OTP" }
+  { "sasl_realm" = "realm" }
+  { "sasl_authcid" = "authcid" }
+  { "sasl_authzid" = "dn:cn=annonymous,dc=example,dc=net" }
+  { "sasl_secprops" = "noanonymous,noplain,minssf=0,maxssf=2,maxbufsize=65535" }
+  { "sasl_canonicalize" = "yes" }
+  {  }
+  { "#comment" = "Kerberos authentication options" }
+  { "krb5_ccname" = "ccname" }
+  {  }
+  { "#comment" = "Search/mapping options" }
+  {  }
+  { "#comment" = "Specifies the base distinguished name (DN) to use as search base." }
+  { "base" = "dc=people,dc=example,dc=com" }
+  { "base" = "dc=morepeople,dc=example,dc=com" }
+  { "base"
+    { "alias" = "dc=aliases,dc=example,dc=com" }
+  }
+  { "base"
+    { "alias" = "dc=morealiases,dc=example,dc=com" }
+  }
+  { "base"
+    { "group" = "dc=group,dc=example,dc=com" }
+  }
+  { "base"
+    { "group" = "dc=moregroup,dc=example,dc=com" }
+  }
+  { "base"
+    { "passwd" = "dc=users,dc=example,dc=com" }
+  }
+  {  }
+  { "#comment" = "Specifies the search scope (subtree, onelevel, base or children)." }
+  { "scope" = "sub" }
+  { "scope"
+    { "passwd" = "sub" }
+  }
+  { "scope"
+    { "aliases" = "sub" }
+  }
+  {  }
+  { "#comment" = "Specifies the policy for dereferencing aliases." }
+  { "deref" = "never" }
+  {  }
+  { "#comment" = "Specifies whether automatic referral chasing should be enabled." }
+  { "referrals" = "yes" }
+  {  }
+  { "#comment" = "The FILTER is an LDAP search filter to use for a specific map." }
+  { "filter"
+    { "group" = "(objectClass=posixGroup)" }
+  }
+  {  }
+  { "#comment" = "This option allows for custom attributes to be looked up instead of the default RFC 2307 attributes." }
+  { "map"
+    { "passwd"
+      { "homeDirectory" = "\"${homeDirectory:-/home/$uid}\"" }
```

Comment:
Should quotes be removed here if present? The `Quote` module has lots of helpers for this, you can use things like `Quote.any_opt` to support store a value and optionally remove quotes if there are either double or single quotes.